Source/ZXing.Net.Mobile.Android/ZXing.Net.Mobile.Android.csproj

```diff
@@ -12,7 +12,10 @@
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <RootNamespace>ZXing.Mobile</RootNamespace>
     <AssemblyName>ZXingNetMobile</AssemblyName>
-    <TargetFrameworkVersion>v4.0.3</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v6.0</TargetFrameworkVersion>
```

Comment:
Google Play Service SDK's requires a Target Framework of at least Android 4.1 (API Level 16) to compile.
Is it ok to set latest paltform sdk or 4.1 is more preferable?